src/kubernetes-ingress-controller/guides/using-tcpingress.md

```diff
@@ -152,21 +203,87 @@ spec:
       serviceName: tcp-echo
       servicePort: 2701
 " | kubectl apply -f -
+```
+{% endnavtab %}
+{% navtab Response %}
+```text
 tcpingress.configuration.konghq.com/echo-plaintext created
 ```
+{% endnavtab %}
+{% endnavtabs %}
+{% endnavtab %}
+{% navtab Gateway APIs %}
+{% navtabs codeblock %}
+{% navtab Command %}
+```bash
+echo "apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: TCPRoute
+metadata:
+  name: echo-plaintext
+spec:
+  parentRefs:
+  - name: kong
+  rules:
+  - backendRefs:
+    - name: tcp-echo
+      port: 9000
+" | kubectl apply -f -
+```
+{% endnavtab %}
 
-Here we are instructing Kong to forward all traffic it receives on port
-9000 to `tcp-echo` service on port 2701.
+{:.note}
+> v1alpha2 TCPRoutes do not support separate proxy and upstream ports. Traffic
+> is redirected to `2701` upstream via Service configuration.
 
-Once created, we can see the IP address at which this is available:
+{% navtab Response %}
+```text
+tcproute.gateway.networking.k8s.io/echo-plaintext created
+```
+{% endnavtab %}
+{% endnavtabs %}
+{% endnavtab %}
+{% endnavtabs %}
 
-```shell
-$ kubectl get tcpingress
+This configuration instructs {{site.base_gateway}} to forward all traffic it
+receives on port 9000 to `tcp-echo` service on port 2701.
+
+## Test the configuration
```

Comment:
IMO using level 3 heading to clear that the "Test of Configuration" section is to test the configurations applied in the "Route TCP traffic by port" would be better. Currently we can see to "Test of Configuration" on the same level of the nav bar. Do you think it is better to use level 3 heading here, or use different titles to show that they are not testing the same configuration?